docker/courtlistener/docker-compose.yml

```diff
@@ -80,6 +80,8 @@ services:
     volumes:
       - ${CL_POSTGRES_RUN_DIR:-/var/run/postgresql}:/var/run/postgresql
       - ${CL_BASE_DIR:-../../}:/opt/courtlistener
+      - ${CL_BASE_DIR:-../../}/docker/postgresql/cl-postgres.crt:/root/.postgresql/root.crt
+      - ${CL_BASE_DIR:-../../}/docker/elastic/cl-es.crt:/run/secrets/cl-es.crt
```

Comment:
`elastic/cl-es.crt` is being used here, instead of `elastic/ca.crt`, because using with `ca.crt` yields this warning with verification on:

```
TLS error caused by: TlsError(TLS error caused by: SSLError([SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: CA cert does not include key usage extension (_ssl.c:1020))), Retrying in 10 seconds...
```

Reply:
I found the problem. It appears that the CA file didn't have the correct extensions. In the past, this wasn't an issue since the verification process didn't check extensions, but in recent versions of `elastic_transport`, this verification has become more strict.

I believe it works using `cl-es.crt` instead of` ca.crt` because it matches the exact certificate in `cl-es`, but it's not performing proper verification using the CA.

I've regenerated the certificates using a proper CA and `elasticsearch-certutil`. 

Which now contains:

```
X509v3 extensions:
            X509v3 Basic Constraints: critical
                CA:TRUE
            X509v3 Key Usage: critical
                Certificate Sign, CRL Sign
```

You can use these files to replace the ones in /docker/elastic:

[ca.crt.txt](https://github.com/user-attachments/files/17708558/ca.crt.txt)
[cl-es.crt.txt](https://github.com/user-attachments/files/17708559/cl-es.crt.txt)
[cl-es.key.txt](https://github.com/user-attachments/files/17708560/cl-es.key.txt)

And use:
ELASTICSEARCH_CA_CERT=/run/secrets/ca.crt

Instead of:
ELASTICSEARCH_CA_CERT=/run/secrets/cl-es.crt